package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "scripts": {"css": "npx tailwindcss -i ./views/application.css -o ./public/css/application.css --watch" },
```

Comment:
can you format this a bit nicer? "css" should be on a new line

Reply:
resolved